docs/product/issues/issue-details/replay-issues/rage-clicks.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Rage Click Issues
+sidebar_order: 40
+description: "Learn about Session Replay rage click issues."
+---
+
+If you've enabled [Session Replay](/product/explore/session-replay/), you'll be able to see rage click issues on the [**Issues**](https://sentry.io/orgredirect/organizations/:orgslug/issues/) page in Sentry. Rage clicks are a series of consecutive clicks on the same unresponsive page element. They are a strong signal of user frustration and most likely deserve your attention.
+
+![Rage click issue](./img/Rage-click-issue-in-issue-stream.png)
+
+## Prerequisites for Seeing Rage Clicks
+
+In order to see rage clicks in your issue stream on the **Issues** page, your organization needs to:
+
+- Be sending [Session Replay events](/product/explore/session-replay/getting-started/)
+- Enable the JavaScript SDK (or framework-specific bundle), version 7.60.1 or higher
+
+<Note>
+While you can enable **Session Replay** with JavaScript SDK version 7.27.0, or higher, you'll need to have version 7.60.1 or higher in order to be able to see **rage click issues**.
+</Note>
+
+## Detection Criteria
+
+"Slow clicks" (also called "dead clicks") are only detected on `<button>`, `<input>`, and `<a>` elements that don't lead to updates to the DOM or a page scroll within 7 seconds). When the user clicks on one of these elements 3 or more times within that 7 second time frame, it indicates frustration, and the SDK registers a "rage click".
+
+You can configure Replay to ignore specific selectors if you are seeing <PlatformLink to="/session-replay/troubleshooting/#why-am-i-seeing-too-many-or-too-few-rage-clicks">too many rage clicks</PlatformLink> reported. For example, you might want to ignore clicks on "Print" or "Download" buttons.
```

Comment:
I think we should change this to a regular link so that it doesn't present us with the option to choose a language (where the only option is JS).
```suggestion
You can configure Replay to ignore specific selectors if you are seeing [too many rage clicks](https://docs.sentry.io/platforms/javascript/session-replay/troubleshooting/#why-am-i-seeing-too-many-or-too-few-rage-clicks) reported. For example, you might want to ignore clicks on "Print" or "Download" buttons.
```